cli/config/config.go

```diff
@@ -58,6 +60,20 @@ type AppOpts struct {
 	// DataDir is the directory where all the instance artifacts
 	// are stored.
 	DataDir string `mapstructure:"data_dir"`
+	// BinDir is the directory where all the binary files
+	// are stored.
+	BinDir string `mapstructure:"bin_dir"`
+	// IncludeDir is the directory where all the header files
+	// are stored.
+	IncludeDir string `mapstructure:"inc_dir"`
+}
+
+// RepoOpts is a struct used to store paths to local files.
+type RepoOpts struct {
```

Comment:
Discussed about Install and Rocks names at friday meeting. About LocalRepo, I don't see a point all other are named name + Opts, so it would be strange to change that strategy for Repo.